difftest/dpi_t1rocketemu/src/drive.rs

```diff
@@ -317,20 +317,10 @@ pub(crate) struct Driver {
 
 impl Driver {
   pub(crate) fn new(scope: SvScope, args: &OnlineArgs<'_>) -> Self {
-    let mut initmem = vec![0; RAM_SIZE as usize];
-    let (e_entry, _fn_sym_tab) =
-      Self::load_elf(Path::new(&args.elf_file), &mut initmem).expect("fail creating simulator");
-
-    let (addr_space, exit_flag) = match args.dramsim3 {
-      Some((cfg_path, run_path)) => {
-        let mem = RegularMemory::with_content_and_model(initmem, cfg_path, run_path);
-        create_emu_addrspace_with_mem(mem)
-      }
-      None => {
-        let mem = RegularMemory::with_content(initmem);
-        create_emu_addrspace_with_mem(mem)
-      }
-    };
+    let dram_model = args.dramsim3.map(|(cfg_path, run_path)| DRAMModel::new(cfg_path, run_path));
```

Comment:
Change dramsim3 to be required. no more optional.

Reply:
Done